src/app/fddev/configure/genesis.c

```diff
@@ -38,14 +39,43 @@ init( config_t * const config ) {
   char vote[ PATH_MAX ];
   FD_TEST( fd_cstr_printf_check( vote, PATH_MAX, NULL, "%s/vote-account.json", config->scratch_directory ) );
 
+  char initial_accounts[ PATH_MAX ];
+  FD_TEST( fd_cstr_printf_check( initial_accounts, PATH_MAX, NULL, "%s/initial-accounts.yaml", config->scratch_directory ) );
+
+  FILE * f = fopen( initial_accounts, "w" );
+  FD_TEST( f );
+  for( ulong i=0UL; i<128UL; i++ ) {
+    /* Fund the first 128 accounts with some lamports for benchmarking... temporary hack */
+    uchar privkey[ 64 ] = {0};
+    privkey[ 0 ] = (uchar)i;
+    uchar pubkey[ 64 ];
+    fd_sha512_t sha[1];
+    fd_sha512_join( fd_sha512_new( sha ) );
+    fd_ed25519_public_from_private( pubkey, privkey, sha );
+
+    char pubkey_encoded[ FD_BASE58_ENCODED_32_SZ ];
+    fd_base58_encode_32( pubkey, NULL, pubkey_encoded );
+    FD_TEST( fprintf( f, "%s:\n", pubkey_encoded ) >= 0 );
+    FD_TEST( fprintf( f, "  balance: 500000000\n" ) >= 0 );
+    FD_TEST( fprintf( f, "  owner: 11111111111111111111111111111111\n" ) >= 0 );
+    FD_TEST( fprintf( f, "  data: ~\n" ) >= 0 );
```

Comment:
`data:\n` worked for me in the past. Does this also make it empty? I can't remember if the field is base64 encoded?

Reply:
I think so? This is what I got from reading the Solana parsing code.